featurebyte/service/feature_table_cache.py

```diff
@@ -475,6 +476,7 @@ async def create_or_update_feature_table_cache(
                         column_names=[InternalName.TABLE_ROW_INDEX.value] + request_column_names,
                     ),
                     exists=True,
+                    retry=True,
```

Comment:
Is there a reason retry is needed here?

Reply:
without this change I often run into this error for Spark source type:

```
pyhive.exc.OperationalError: TExecuteStatementResp(status=TStatus(statusCode=3, infoMessages=['*org.apache.hive.service.cli.HiveSQLException:Error running query: io.delta.exceptions.ProtocolChangedException: The protocol version of the Delta table has been changed by a concurrent update. This happens when multiple writers are writing to an empty directory. Creating the table ahead of time will avoid this conflict. Please try the operation again.\
```